examples/get_started/04_table_inputs.py

```diff
@@ -0,0 +1,146 @@
+"""
+4. PyGMT I/O: Table inputs
+==========================
```

Comment:
I/O means Input/Output, but this tutorial is only on inputs :slightly_smiling_face: Will the 'Output' part be added as a separate page? Or do we want multiple parts like:

- 4.1 PyGMT I/O: Table inputs
- 4.2 PyGMT I/O: Table outputs
- 4.3 PyGMT I/O: Grid inputs
- 4.4 PyGMT I/O: Grid outputs

Reply:
Yes, that's my plan, but I may prefer to the following order:

4.1 PyGMT I/O: Table inputs
4.2 PyGMT I/O: Grid inputs
4.3 PyGMT I/O: Table outputs
4.4 PyGMT I/O: Grid outputs